packages/rpc-core/test/e2e/state.test.ts

```diff
@@ -2,10 +2,20 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Balance, Bytes, Metadata, Moment } from '@polkadot/types';
+import fs from 'fs';
+import path from 'path';
+
+import { Balance, Bytes, Hash, Metadata, Moment, StorageData, StorageKey } from '@polkadot/types';
+import { Abi } from '@polkadot/api-contract';
 import storage from '@polkadot/storage/static';
 import WsProvider from '@polkadot/rpc-provider/ws';
 
+import ApiPromise from '@polkadot/api/promise/Api';
```

Comment:
(nit) Alphabetical ordering (yes, these are tests, but in this case we are starting to do a lot of imports)